core/src/io/anuke/mindustry/ui/Links.java

```diff
@@ -38,7 +58,13 @@ private static void createLinks(){
         public LinkEntry(String name, String link, Color color){
             this.name = name;
             this.color = color;
-            this.description = Core.bundle.getNotNull("link." + name + ".description");
+            String key = "link." + name + ".description";
+            String desc = Core.bundle.getOrNull(key);
+            if(desc == null){
+                missingLocales.add(key+"\n"); // <- LINE 62
+                desc = key;
+            }
+            this.description = desc;
```

Comment:
You can add line breaks before and after the block to clarify the code like this:
```java
            this.color = color;

            String key = "link." + name + ".description";
            String desc = Core.bundle.getOrNull(key);
            if(desc == null){
                missingLocales.add(key+"\n"); // <- LINE 62
                desc = key;
            }
            this.description = desc;

            this.link = link;
```
You can also add more line break into the new block.
``` java
            String key = "link." + name + ".description";
            String desc = Core.bundle.getOrNull(key);

            if(desc == null){
                missingLocales.add(key + "\n"); // <- LINE 62
                desc = key;
            }

            this.description = desc;
```